### PR TITLE
Split up and reorganise dev tutorials

### PR DESCRIPTION
**Pull request by hexylena**
image

### CODE REVIEW

**Comment by jmchilton** [comment]
Very nice improvement!

**Comment by shiltemann** [comment]
Nice! (I just removed the superfluous "Other" section if it is empty)

**Review by shiltemann — APPROVED** [review]

**Review by lecorguille — APPROVED** [review]

File: topics/dev/metadata.yaml [hunk 1]
```diff
@@ -11,3 +11,23 @@ maintainers:
   - hexylena
 
 gitter: galaxyproject/dev
+
+subtopics:
```
Comment (lecorguille) [inline topics/dev/metadata.yaml:15]: Really good idea.
However, I shouldn't have ordered like that but more like:
core -> API -> dependencies -> tooldev -> viz -> advanced

**Comment by shiltemann** [comment]
\o/ Thanks @hexylena and @lecorguille!

**Pull request merged**